src/components/portrait/portrait.pw.tsx

```diff
@@ -47,11 +50,20 @@ const portraitSizes = PORTRAIT_SIZES.map((size) => [
 ]);
 
 test.describe("Prop checks for Portrait component", () => {
+  ["SPM", "JM", "AR", "MJ"].forEach((passInitials) => {
```

Comment:
Yeah i've covered this in a unit test but I don't mind adding a Playwright test too

Reply:
Actually, a unit test should be enough for this for now. If we discover in future, that some sort of user interaction can cause more than three to appear in the browser, then it would warrant a Playwright test.

Reply:
Just added and amended some playwright tests anyways to test the functionality, can't hurt 👍